requirements.txt

```diff
@@ -1,19 +1,19 @@
-beautifulsoup4==4.7.1
-beautifultable==0.7.0
-boto==2.49.0
-boto3==1.9.88
-botocore==1.12.116
-click==6.7
-coverage==4.5.1
-coveralls==1.3.0
-docker==3.6.0
-flake8==3.0.4
-lxml==4.2.1
```

Comment:
@RohanSreelesh Why is lxml removed as a dependency in this commit?

Reply:
My build wasnt installing , then @yashdusing on gitter suggested removing this from requirements and setup.py after which I was able to install it successfully 
The error can be found here https://imgur.com/g4wefJP

Reply:
Some development libraries are missing in the python package related to lxml, thus the package was not installing. You can find more details [here](https://stackoverflow.com/questions/33785755/getting-could-not-find-function-xmlcheckversion-in-library-libxml2-is-libxml2). 

We cannot remove it from the requirements as it would break some other parts of the code(for example, the build fails as the lmxl based tests are not passing). 

The issue exists on windows due to the version of lxml (please refer to the second answer on the link about. maybe you could update the version of lxml in requirements.txt and try reinstalling on windows to resolve the issue.)

Reply:
I understand , I had done the solutions given in the link including the second answer but the build still didnt work for me unfortunately. I'll add it back

Reply:
I added lxml back into requirements and setup.py but the build is still failing @pushkalkatara

Reply:
The build is failing due to some more issues. You can get all the details [here](https://travis-ci.org/Cloud-CV/evalai-cli/builds/622555026?utm_source=github_status&utm_medium=notification).

1. flake8 error
2. the test outputs don't match (you need to update the tests also.)